gdx/src/com/badlogic/gdx/AbstractInput.java

```diff
@@ -3,18 +3,36 @@
 
 import com.badlogic.gdx.utils.IntSet;
 
+/** Abstract implementation of the Input interface, providing functionalities for handling key presses, including checking if keys
+ * are currently pressed or have just been pressed. This class is intended to be extended by platform-specific input
+ * implementations. */
+
 public abstract class AbstractInput implements Input {
```

Comment:
If Input already has JavaDocs for a method, you generally shouldn't make duplicate JavaDocs in each subclass, unless something is different in that subclass that invalidates the parent docs.